src/EventSubscriber/RequestSubscriber.php

```diff
@@ -263,4 +274,35 @@ protected function constructQuerySpan($connection, array $query) {
     return $span;
   }
 
+  /**
+   * Add options to pass to the PHP Agents.
+   *
+   * @return array
+   *   An array of options to pass to the PHP Agent. Ie. tags.
+   */
+  protected function getRequestOptions() {
+    $options = [];
+
+    // Fetch the current path.
+    $route_object = $this->routeMatch->getRouteObject();
+    $path = $route_object->getPath();
+    $route_options = $route_object->getOptions();
+
+    // If this is an admin page, add a custom variable to denote that.
+    if (isset($route_options['_admin_route'])) {
+      $options['tags']['admin_page'] = TRUE;
```

Comment:
**CHANGE REQUEST**

Let's name this tag just `admin` for consistency and because `_page` is unnecessary; it's like naming the rest `commerce_page` or `node_page`.